cross-chain/arbitrum/deploy_l2/11_transfer_arbitrum_tbtc_ownership.ts

```diff
@@ -0,0 +1,14 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
The prefix in `21_authorize_wormhole_gateway_as_minter.ts` suggests it happens after this step. 

What's more, it has `func.runAtTheEnd = true` so it happens after the ownership is transferred. I am confused why the deployment works at all. The transaction should fail on authorizing the gateway.

Reply:
see https://github.com/keep-network/tbtc-v2/pull/554#discussion_r1151787156